cypress/e2e/0-api/receivers_api.cy.js

```diff
@@ -0,0 +1,183 @@
+import { isIlcCapable } from '../../support/device_helpers';
+
+describe('receivers API', () => {
+  const serialNumbers = [];
+  let start;
+  let end;
+  before(function() {
+    cy.reqInfo().then(response => {
+      const { body } = response;
+      const serialNumber = body.serialNumber;
+      if (!isIlcCapable(serialNumber)) {
+        this.skip();
```

Comment:
I think you can give a message?  
```suggestion
        this.skip(`Not ILC capable: s/n: ${serialNumber}`);
```